SparkPost/2web2ui
# AC-1977 - Global tracking for all forms in snippets and blocklist

AC-1977 - Global tracking for all forms in snippets and blocklist

### What Changed

- Added condition to prevent undefined fields from making into formData
- Added a cypress test for snippets create page

Notes - 
- In Blocklist Watchlist Add Page, the api error won't show up in segment data since its an api error
- In Snippets Create and EditPage, Content Editor Field error won't show up in formData, since its a error in label (reduxForm field)

### How To Test

- Check for form submissions on Snippets.
- Check for form submissions on Blocklist page

### To Do
